authentik/sources/oauth/types/discord.py

```diff
@@ -43,3 +33,10 @@ class DiscordType(SourceType):
     authorization_url = "https://discord.com/api/oauth2/authorize"
     access_token_url = "https://discord.com/api/oauth2/token"  # nosec
     profile_url = "https://discord.com/api/users/@me"
+
```

Comment:
We could optionally fetch the servers the user is in and provide that as group data since that seems to be pretty common use-case?

Reply:
As discussed, we won't because we would automatically create those groups on update, which may not be desirable.